AndelaOSP/art-backend
# #159022584 Implement AssetSpecs unique validation

## What does this PR do?
 - It Implements unique together validation on  AssetSpecs Model

## Description of tasks to be completed
- write tests asset specs unique validation
- add unique_together meta field on AssetSpecs Model
- write a custom unique validator

## How can this PR be manually tested
- make a `post` api call on `http://127.0.0.1:8000/api/v1/asset-specs` using postman or tools like curl, see screenshot below.
- make another `post` request with the same field values, you will see an error response as shown below

## Relevant Pivotal Tracker Stories
[#159022584](https://www.pivotaltracker.com/story/show/159022584)

## ScreenShots
<img width="1191" alt="screen shot 2018-07-13 at 11 02 48 am" src="https://user-images.githubusercontent.com/31279497/42686822-fe5e77c2-868e-11e8-8ebd-06240cb8c8dd.png">
